transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringEventLoop.java

```diff
@@ -52,9 +52,11 @@
     private final FileDescriptor eventfd;
 
     private final IovArrays iovArrays;
+    // The maximum number of bytes for an InetAddress / Inet6Address
+    private final byte[] inet4AddressArray = new byte[4];
+    private final byte[] inet6AddressArray = new byte[16];
 
     private long prevDeadlineNanos = NONE;
-    private boolean pendingWakeup;
```

Comment:
We may want to leave this here since I think some similar logic will be needed to what this is used for with epoll - specifically we need to do a final blocking read upon event loop shutdown before closing the eventfd if this flag is set, otherwise the corresponding "wakeup" write could be made after the fd has been reassigned to something else.

Reply:
@njhill can you do a pr ?

Reply:
Let me pull this one in first and we can add this stuff back when needed.

Reply:
@normanmaurer @1Jo1 I've now opened a PR for this: #10615